### ISSUE TITLE
[pyos][docs] Versions and documentation

### ISSUE THREAD

**Issue by szhorvat** [python-graphblas/python-graphblas#471]
It would be nice to indicate in the documentation, especially the API documentation, what specific version the docs are for.

It would also be useful to be able to switch to the docs of older versions.

Part of: https://github.com/pyOpenSci/software-submission/issues/81

**Comment by eriknw** [comment]
Agreed. @jim22k any reason why we don't do this?

**Comment by jim22k** [comment]
No specific reason other than it was easier to focus on writing current docs. No thought was given to how those versions of the docs are maintained over time.

I agree we should do this moving forward.

**Comment by eriknw** [comment]
Almost done.

I added an automation rule on readthedocs to automatically activate non-prerelease versions when we do new releases in the future.

I also activated docs versions going back to `2022.11.1`, b/c that's the last version that built today without any issues.

The version and available versions now appear in the [flyout menu](https://docs.readthedocs.io/en/stable/flyout-menu.html) in the lower left in all pages except the initial landing page.

**The only remaining thing to do before closing this issue is to somehow show the version on the landing page (or decide not to).**
